<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VINAY" ignored="0" name="TestB1" tests="1" failures="0" timestamp="28 Sep 2018 08:13:37 GMT" time="24.464" errors="0">
  <testcase name="@BeforeMethod openApplication" time="24.258" classname="generic.BaseTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at generic.BaseTest.openApplication(BaseTest.java:31)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod openApplication -->
  <testcase name="@AfterMethod closeApplication" time="0.009" classname="generic.BaseTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at generic.BaseTest.closeApplication(BaseTest.java:48)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod closeApplication -->
  <testcase name="testA" time="0.0" classname="script.TestDemo">
    <skipped/>
  </testcase> <!-- testA -->
</testsuite> <!-- TestB1 -->
